R/FilteredData.R

```diff
@@ -795,12 +794,31 @@ FilteredData <- R6::R6Class( # nolint
             active_datanames = active_datanames
           )
 
-          shiny::observeEvent(self$get_filter_state(), {
-            if (length(self$get_filter_state()) == 0) {
+          shiny::observeEvent(input$minimise_filter_overview, {
+            shinyjs::toggle("filters_overview_contents")
+          })
+
+          shiny::observeEvent(input$minimise_filter_active, {
+            shinyjs::toggle("filter_active_vars_contents")
+            shinyjs::toggle("filters_active_count")
+            toggle_icon(session$ns("minimise_filter_active"), c("fa-angle-right", "fa-angle-down"))
+            toggle_title(session$ns("minimise_filter_active"), c("Restore panel", "Minimise Panel"))
+          })
+
+          shiny::observeEvent(private$get_filter_count(), {
+            if (private$get_filter_count() == 0) {
               shinyjs::hide("remove_all_filters")
             } else {
               shinyjs::show("remove_all_filters")
             }
```

Comment:
Please rather use `condition` argument instead
```suggestion
            shinyjs::hide("remove_all_filters", condition = private$get_filetr_count() == 0)
```